api/SVGStopElement.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "SVGStopElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGStopElement",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
```

Comment:
Please sort the `support` block alphabetically (see issue #398 and PR #1882), this applies to all your other PRs.